client/my-sites/email/email-management/style.scss

```diff
@@ -461,108 +471,127 @@
 		}
 	}
 
-	.email-plan-mailboxes-list__mailbox-list {
-		border: solid 1px var(--color-border-subtle);
-		border-radius: 4px;
+	/* Mailboxes */
+	.email-plan-mailboxes-list {
+		&__mailbox-list {
+			border: solid 1px var(--color-border-subtle);
```

Comment:
This border looks a bit off on mobile. What do you think of removing it there?

| With border  | Without border |
| ------------- | ------------- |
| ![image](https://github.com/user-attachments/assets/9d42a20d-a4ac-474d-9d5d-b1ad259254f4)  | ![image](https://github.com/user-attachments/assets/357d6b97-691e-4d29-9440-f9acc2bdde13)   |